pages/api/upvotes.ts

```diff
@@ -0,0 +1,32 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import axios from "axios";
+import { load } from "cheerio";
+
+type Data = {
+  upvotes: number;
+};
+
+export default async (req: NextApiRequest, res: NextApiResponse<Data>) => {
+  try {
+    console.log("count");
+    const { data } = await axios.get(
+      "https://www.producthunt.com/products/capybara-api"
+    );
+    const $ = load(data);
+    const upvotes = Number(
+      $("div")
```

Comment:
Feels wrong and inefficient using cheerio with next API routes let alone react in general, is there no public API for this?

Reply:
there is but it requires token and its graphql

Reply:
bruhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh